tutorials/introductory/usage.py

```diff
@@ -467,7 +498,7 @@ def my_plotter(ax, data1, data2, param_dict):
 fig.colorbar(co, ax=axs[0, 1])
 axs[0, 1].set_title('contourf()')
 
-pc = axs[1, 0].imshow(Z**2 * 100, cmap='plasma',
+pc = axs[1, 0].imshow(Z**2 * 100, cmap='plasma', aspect='auto',
```

Comment:
I don't know if we should do this.  A major difference between `imshow` and `pcolormesh` are their different default aspect ratios.

Reply:
I didn't mean to change it for content or convey any meaning with it; it just looked a bit awkward with the left justified imshow plot in the subplot grid and the title going over the y-axis:
**OLD:**
![grafik](https://user-images.githubusercontent.com/19879328/144023532-6229fdd9-3b60-4251-84f3-a3c2307f7b8a.png)
**NEW:**
![grafik](https://user-images.githubusercontent.com/19879328/144023617-737fa34d-b9a5-4577-b50a-2bde59627bc6.png)
  
<br>  

So it was just for esthetics - no problem to revert it if you prefer.

Reply:
I agree it looks better - but I think folks should realize that this is happening with `imshow` by default.

Reply:
so should I 
- keep `aspect='auto'`
- leave the default 1:1 with the small image under the large title or 
- increase the number of points in the x linspace to 152?

I mean in the end this section illustrates the use of color as a third dimension, not the details of `imshow`.

Reply:
It illustrates the various methods to display the third dimension, and is explicitly meant to show the difference between them.  I'd prefer to keep the aspect ratio as the default.

Reply:
OK, will revert it.